examples/seismic/tutorials/16_variable_z_topography.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        for i in range(f.space_order//2):
</pre><p>Use Mathias' free-surface with a <code>SubDomain</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>